product_manufacturer/tests/test_product_manufacturer.py

```diff
@@ -0,0 +1,131 @@
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+
+from odoo.tests.common import TransactionCase
+
+
+class TestProductManufacturer(TransactionCase):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.env = cls.env(context=dict(cls.env.context, tracking_disable=True))
```

Comment:
FYI, Odoo has done it in a common test class now: https://github.com/odoo/odoo/blob/17.0/odoo/addons/base/tests/common.py#L21